lib/turbo_tests/windows.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module TurboTests
+  class Runner
+    private
+
+    def start_subprocess(env, extra_args, tests, process_id, record_runtime:)
+      if tests.empty?
+        @messages << {
+          type: "exit",
+          process_id: process_id,
+        }
+      else
+        require "securerandom"
+        env["RSPEC_FORMATTER_OUTPUT_ID"] = SecureRandom.uuid
+        env["RUBYOPT"] = ["-I#{File.expand_path("..", __dir__)}", ENV["RUBYOPT"]].compact.join(" ")
+
+        command_name = Gem.win_platform? ? [Gem.ruby, "bin/rspec"] : "bin/rspec"
```

Comment:
Good point too! Using `bin/rspec` directly is quite specific to our environment. I guess we could detect whether there's a `bin/rspec` file relative to the cwd, and otherwise use `bundle exec rspec`?

Reply:
> I guess we could detect whether there's a `bin/rspec` file relative to the cwd, and otherwise use `bundle exec rspec`

Will Windows correctly handle this if we append `$PWD/bin` to ENV["PATH"] instead of detecting `bin/rspec`? (I'm new to Bundler internals. Below is a hacky pseudo-code.)

```ruby
env["PATH"] << ":#{File.expand_path("bin", __dir__)}"

command_name = ENV["BUNDLE_BIN_PATH"] ? [ENV["BUNDLE_BIN_PATH"], "exec", "rspec"] : "rspec"
```

Reply:
I think it would handle it fine, but I'm not sure it's a good idea since it would have global side effects. Users have other scripts in `bin/` and maybe won't expect turbo_tests to automatically set things up so that those other scripts start getting selected when their tests run processes? Not sure.

Maybe we can start leaving things as they are regarding building the command name for now. I could configure `$LOAD_PATH` on our side before launching turbo_tests so that our binstub gets properly loaded.